app/controllers/stages_controller.rb

```diff
@@ -1,13 +1,18 @@
 require 'open-uri' # needed to fetch from img.shields.io using open()
 
 class StagesController < ApplicationController
+  include CurrentProject
```

Comment:
As mentioned, all this situations have been addressed in the following PR: #567 :)